tszip/cli.py

```diff
@@ -134,8 +142,13 @@ def run_decompress(args):
         if not file_arg.endswith(args.suffix):
             exit(f"Compressed file must have '{args.suffix}' suffix")
         infile = pathlib.Path(file_arg)
-        outfile = pathlib.Path(file_arg[: -len(args.suffix)])
-        check_output(outfile, args)
+        if args.stdout:
+            args.keep = True
+            # NOTE: this is likely not compliant across different systems
+            outfile = pathlib.Path("/dev/stdout")
```

Comment:
We should be able to use ``sys.stdout``, right?

Reply:
This certainly works when running in the command line (e.g. `tszip -d -c tests/files/1.0.0.trees.tsz > test.trees`). However when running in the `pytest` suite it seems to complain substantially about this (see image attached). I think the crux is that `sys.stdout` is not the same as a standard file-handle and does not have the `fileno` operation? 

I find it odd that this only breaks in the `pytest` environment though and not directly on my commandline. 
 
<img width="920" alt="Screen Shot 2021-11-04 at 8 56 01 AM" src="https://user-images.githubusercontent.com/2427335/140317384-fc508307-11b5-499f-825a-73ec88e74702.png">

Reply:
Ah, yuck, this is just an artefact of pytest. Pytest is capturing stdout and it's replacing it with a "not real" file.

We do something similar in msprime for the ``msp ancestry`` command, which outputs to stdout by default. Perhaps there's something in the ``capture_output`` [function](https://github.com/tskit-dev/msprime/blob/e00bb3917dd65d24e2587228bfcfa836f74f6622/tests/test_cli.py#L41) which helps?